Dart/src/com/jetbrains/lang/dart/DartPluginCapabilities.java

```diff
@@ -0,0 +1,21 @@
+package com.jetbrains.lang.dart;
+
+import com.intellij.util.containers.HashMap;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+
+/**
+ * A queryable class about the capabilities of the Dart plugin.
+ */
+public class DartPluginCapabilities {
+  public static Map<String, Boolean> capabilities = new HashMap<>();
```

Comment:
may be make it `private`?

Also, may be a `Set` is enough? Set contains -> feature supported, doesn't contain -> not supported.